api/peripheral_api/routes/Printer.js

```diff
@@ -27,7 +27,24 @@ AWS.config.update({
 });
 
 router.get('/healthCheck', async (req, res) => {
-  return res.sendStatus(OK);
+/*
+ * How these work with Quasar:
+ * https://github.com/SCE-Development/Quasar/wiki/How-do-Health-Checks-Work%3F
+ */
+  if (process.env.NODE_ENV !== 'production') {
+    return res.sendStatus(OK);
+  }
```

Comment:
we dont use this anymore see PR #1109 for details

Reply:
we still have `ARG NODE_ENV` though?